src/System.Web.Services.Description/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0008</DiagnosticId>
+    <Target>T:System.Web.Services.Configuration.XmlFormatExtensionAttribute</Target>
+    <Left>ref/netstandard2.0/System.Web.Services.Description.dll</Left>
```

Comment:
@ericstj, do you know what's going on with these three? As far as I can tell, these are correct. The CP0008 error means a base interface was removed from the interface hierarchy from one of the compared sides. Is this caused by Attribute no longer implementing the interface `_Attribute`? Is this an API compat tool bug as the interface is defined on Attribute, which is external to these packages so isn't a difference between WCF Client packages.

Reply:
I'll have a closer look at this and the others.  The logged messages give a bit more context - @ViktorHofer has improved human readability of the suppression files in later 10.0 releases.

A lot of times you still need to look at the metadata to understand what's going on.  I'll have a review and add detail by mid-day tomorrow.